recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "arcticdb" %}
-{% set version = "4.3.0rc1" %}
+{% set version = "4.3.0rc2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/man-group/ArcticDB/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 8f9d65c37bc586443f88aa525c0f0a66da65b0b38a3f04a38bebd32e85dd22a2
+  sha256: 32e8cd339e0c8e22f9fbe38a5afd4208e6eb2cec4994d6561e47be6998d1fbc2
```

Comment:
```suggestion
  sha256: 21a96ba6ef5d690b23b67a1b7903e311319194aaea01b6d307ef4f551e5f9621
```

Reply:
Weird. How did I get the wrong checksum?
Thanks for the update.

Reply:
I think the one reported on GitHub is different than the one of the tarball. I tend to verify it with `sha256sum`.